bitsail-connectors/connector-rocketmq/src/test/resources/bitsail_rocketmq_print.json

```diff
@@ -0,0 +1,31 @@
+{
+  "job": {
+    "common": {
+      "job_type": "STREAMING",
+      "job_plugin_lib_dir": "plugin",
+      "job_plugin_conf_dir": "plugin_conf",
+      "enable_dynamic_loader": true,
+      "instance_id": "1",
+      "internal_instance_id": "1",
+      "extra_properties": {
+        "update-mode": "append"
```

Comment:
Actually I don't understand what this option means, "update-mode". Is it necessary?